common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -38,6 +38,67 @@
 #include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
 #endif
 
+// TODO(wep21): Remove these apis
+//              after they are implemented in ros2 geometry2.
```

Comment:
Did you copy this comment from somewhere?

Reply:
yes, just copy from somewhere. It is originally commented but someone may removed it.
If not needed I will remove it.

Reply:
OK. It's fine as it is.